network_modeling/models/cray-network-architecture.yaml

```diff
@@ -129,6 +138,9 @@ network_v2:
     - lookup_name: ["cdu"]
       shasta_name: "sw-cdu"
       architecture_type: "mountain_compute_leaf"
+    - lookup_name: ["sw-hsn"]
+      shasta_name: "sw-hsn"
+      architecture_type: "slingshot_hsn_switch"
```

Comment:
the architecure_type here needs to match w/ "model" above.  Matches earlier suggestion that model be changed to "slingshot_hsn_switch".  Lookup and shasta_names are correct.

Reply:
This part confused me.  Matching architecture to a model didn't really jive with my brain.